package.json

```diff
@@ -16,18 +16,17 @@
   ],
   "files": [
     "LICENSE",
-    "build",
+    "artifacts",
     "contracts",
     "typings"
   ],
   "scripts": {
-    "build": "truffle compile --all && yarn gen:contract-typings && tsc -b",
-    "clean": "rm -rf build test typings/contracts",
-    "migrate": "truffle migrate --network ganache --reset",
-    "migrate:rinkeby": "truffle migrate --network rinkeby",
-    "test": "tsc -b && truffle test --network ganache build/*.spec.js",
-    "test:benchmark": "tsc -b && truffle test --network ganache build/*.bench.js",
-    "gen:contract-typings": "typechain --target ethers --outDir typings/contracts './build/contracts/*.json'",
+    "build": "hardhat clean && yarn compile && yarn gen:typings",
```

Comment:
Maybe double check that the bytecode doesn't change, because if it did we should do a major version bump.